projects/lib/package.json

```
@@ -30,10 +31,10 @@
     "url": "https://github.com/abacritt/angularx-social-login/issues"
   },
   "dependencies": {
-    "tslib": "^2.0.0"
+    "tslib": "^2.3.1"
   },
   "peerDependencies": {
-    "@angular/common": "^12.0.0",
-    "@angular/core": "^12.0.0"
+    "@angular/common": "^13.2.6",
+    "@angular/core": "^12.2.6"
```

is this intentional? I expected 
```suggestion
    "@angular/core": "^13.2.6"
```

---

Forgot to commit that change, since I was still fixing tests.

Thanks for noticing. Updated it